public/app/features/logs/components/PopoverMenu.tsx

```diff
@@ -80,6 +84,12 @@ export const PopoverMenu = ({
   );
 };
 
+function track(event: string, selectionLength: number) {
+  reportInteraction(`grafana_explore_logs_${event}`, {
+    selectionLength,
+  });
+}
+
```

Comment:
How about we also track `row.datasourceType`?

Reply:
Great idea, added!